frontend/public/components/cloud-services/catalog-source.tsx

```diff
@@ -67,7 +67,9 @@ export const PackageRow: React.SFC<PackageRowProps> = (props) => {
 
 export const PackageList: React.SFC<PackageListProps> = (props) => <List
   loaded={true}
-  data={props.packages.map(pkg => ({...pkg, rowKey: pkg.packageName}))}
+  // TODO(alecmerdler): Adding `metadata` as a hack to get text filter to work until `Package` is a real k8s resource
```

Comment:
We are planning to have an aggregated API server for `packages.operators.coreos.com`, which will make all of this hackery go away.

Reply:
Yeah, was going to suggest we add a new filter to listFilters in list.tsx, but I'm fine with this since it will eventually go away